builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,191 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        3%{?dist}
+
+License:        MIT
+Summary:        Azure IoT Edge Module Runtime
+URL:            https://github.com/azure/iotedge
+
+%{?systemd_requires}
+BuildRequires:  systemd
+BuildRequires:  rust = 1.47.0
+Requires(pre):  shadow-utils
+Requires:       openssl
+Requires:       aziot-identity-service
```

Comment:
We should have a strict requirement for the version of IIS as well. Please do 
`Requires:       aziot-identity-service = 1.2.4-1`

Reply:
The version specification seems to be a limitation here since this package is not hosted on PMC and cannot query to see if that specific version is available. this addition will cause it to fail unless we build IIS at the same time